src/description.cpp

```diff
@@ -875,6 +879,18 @@ void Description::Media::addRTPMap(const Description::Media::RTPMap &map) {
 
 std::vector<uint32_t> Description::Media::getSSRCs() { return mSsrcs; }
 
+std::optional<std::string> Description::Media::getCNameForSsrc(uint32_t ssrc) {
+	for (auto &val : mAttributes) {
+		if (val.find("ssrc:") == 0 && val.find("cname:") != std::string::npos) {
```

Comment:
Just a note, in a future refactoring, it would be worth properly parsing SSRCs and their attributes and storing them in descriptions rather than relying and string lookups every time.

Reply:
Agree.